src/main/webapp/js/instructorFeedbackResults.es6

```diff
@@ -0,0 +1,329 @@
+/* global selectElementContents:false,
+          bindCollapseEvents:false,
+          bindPublishButtons:false,
+          bindUnpublishButtons:false,
+          setStatusMessage:false,
+          showSingleCollapse:false,
+          hideSingleCollapse:false,
+          toggleSingleCollapse:false
+*/
+
+$(document).ready(() => {
+    const participantPanelType = 'div.panel.panel-primary,div.panel.panel-default';
+
+    $('a[id^="collapse-panels-button-section-"]').on('click', (e) => {
+        const isGroupByTeam = document.getElementById('frgroupbyteam').checked;
+        const childPanelType = isGroupByTeam ? 'div.panel.panel-warning' : participantPanelType;
+        const panels = $(e.currentTarget).closest('.panel-success')
```

Comment:
Tell me if you guys want me to revert the `$(e.currentTarget)` part back to `$(this)` and change the arrow function back into a normal one.